doc/api/readline.md

```diff
@@ -369,7 +369,8 @@ changes:
   * `crlfDelay` {number} If the delay between `\r` and `\n` exceeds
     `crlfDelay` milliseconds, both `\r` and `\n` will be treated as separate
     end-of-line input. Default to `100` milliseconds.
-    `crlfDelay` will be coerced to `[100, 2000]` range.
+    `crlfDelay` will be coerced to an Integer no less than `100`, unless it is
+    `Infinity`.
```

Comment:
I would still call it `coerced to a number` instead of `to an integer`. There is no practical difference, and it allows you to drop the `unless it is Infinity` bit (that can still be mentioned, but the way this is written right now it’s a bit confusing). For example:

    `crlfDelay` will be coerced to a number no less than `100`. It can be set to `Infinity`, in which case `\r` followed by `\n` will always be considered a single newline.